### DISCUSSION TITLE
Web page not loading but DNS is working

### DISCUSSION THREAD

**Discussion by js-d-coder** [AdguardTeam/AdGuardHome#7101] [Q&A]
Hi guys, first of all, really appreciate your help on this.

Background: I have been using Adguard Home for more than a year now. And I haven't updated/touched the server configs for many months.

Setup: I am running Adguard home for plain DNS, DoH, Dot and use Nginx, as reverse proxy, for Web page and DoH. DoT and plain DNS (port 53) are directly exposed. Adguard Home version is `0.107.51` (latest as of today) and Nginx version is `1.18.0-6ubuntu14.4`, both running on Ubuntu 22.04.

Problem: The web page is not loading as seen in the image below. But if I copy the Network requests as cUrl and run them in Postman, then the requests work i.e. give response as expected. The problem exists in Firefox and Brave browsers. DNS is working i.e. I can make DNS query via plain DNS and DoH.

![Screenshot_20240626_154018](https://github.com/AdguardTeam/AdGuardHome/assets/23054879/0236222c-05a0-4bc8-8330-3b2a61a5969e)

Adguard home config file: https://pastebin.com/raw/iuK6zvEB  
Nginx config file: https://pastebin.com/raw/hVkhaSSw

Please help me debug this issue. Thank you in advance.

**Comment by ainar-g** [comment]
Do the `curl` queries work with Nginx or with AGH itself directly?  If it's one and not the other, it could be Nginx's limit on the length of the URL/header size, which you'd need to increase in the config.

**Comment by js-d-coder** [comment]
1. The cUrl I use hit nginx. Adguard Home's web page and DoH is never directly accessible.
2. It cannot be the length of the URL because the 'update' endpoint is also failing.
